### ISSUE TITLE
test extra pulls in plone.tiles

### ISSUE THREAD

**Issue by jensens** [plone/plone.restapi#1368]
https://github.com/plone/plone.restapi/blob/42387edcb758396fbc97d0ded59de87d63767ea6/setup.py#L42

But `plone.tiles` is not Plone Core. This should be optional in tests.

**Comment by tisto** [comment]
@jensens ok, the tiles endpoint is deprecated, so this should go away asap.

**Comment by tisto** [comment]
@jensens we would have to cut a plone.restapi release 9 to get rid of the plone.tiles dependency:

https://github.com/plone/plone.restapi/blob/2cbdefe6155c6ec80305b5798b719e002156929f/docs/source/tiles.rst#tiles

I am reluctant to do so for such a trivial thing. I guess an alternative could be removing the test dependency and the tests in the 8 branch if that dependency causes troubles.

**Comment by jensens** [comment]
I do not think this justifies a new release. but the test could have a conditionla import and a `skipIf`, that would be enough.

**Comment by davisagli** [comment]
The tiles endpoint was removed a while ago in https://github.com/plone/plone.restapi/pull/1688
